totalopenstation/formats/sokkia_sdr33.py

```diff
@@ -23,14 +23,14 @@
 
 class FormatParser(Parser):
 
-    def is_point(self, line):
-        if line[2:4] == ('TP'):
+    def is_point(self, line) :
+        if line[2:4] == ('TP') and line[0:2] != '07':
             return True
         else:
             return False
 
     def get_point(self, line):
-        id = int(line[12:20])
+        id = str(line[12:20])
```

Comment:
Failed is due to https://github.com/totalopenstation/totalopenstation/pull/152/commits/3e7eb44335c303bc13c7fe991004b9097e8f2b6c
So maybe adding a try/exception to `ValueError` in `get_point()` function to keep an `int` and have a side effect with a `str`.
```
try :
    id = str(line[12:20])
except ValueError:
    id = int(line[12:20])
```

Reply:
I tryed to do a pullrequest with this modification but doesn't works so I will try to do a new format.
in attachement a file sdr33 exported from sokkia rku 630.  
[sdr33_from_sokkiarku630.zip](https://github.com/totalopenstation/totalopenstation/files/7203890/sdr33_from_sokkiarku630.zip)